recipes/c-blosc/all/test_package/conanfile.py

```diff
@@ -12,6 +13,6 @@ def build(self):
         cmake.build()
 
     def test(self):
-        if not tools.cross_building(self):
+        if not tools.build.cross_building(self, self):
```

Comment:
```suggestion
        if not tools.build.cross_building(self):
```

Reply:
I have fixed this in later script version